scripts/sanitycheck

```diff
@@ -470,10 +470,17 @@ Artificially long but functional example:
         "--test-only", action="store_true",
         help="""Only run device tests with current artifacts, do not build
              the code""")
+
     parser.add_argument(
         "--cmake-only", action="store_true",
         help="Only run cmake, do not build or run.")
 
+    parser.add_argument(
+        "--filter", choices=['buildable', 'runnable', 'all'],
+        default='all',
+        help="Only run cmake, do not build or run.")
```

Comment:
I think this `help` is wrong. I was able to build and run with `--filter buildable`

Reply:
I am a bit confused. Does `--filter buildable` filter anything out or does `all`=`buildable`?

Reply:
all == buildable, remove it from the list above to avoid confusion.